stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -628,4 +638,78 @@ protected void closeInStream() {
       }
     }
   }
+
+  private final class AlluxioPOSIXBenchThread extends BenchThread {
+
+    private RandomAccessFile mRandomAccessFile = null;
+
+    private AlluxioPOSIXBenchThread(BenchContext context, int threadId) {
+      super(context, threadId);
+    }
+
+    @Override
+    @SuppressFBWarnings("BC_UNCONFIRMED_CAST")
+    protected int applyOperation() throws IOException, AlluxioException {
+      if (mRandomAccessFile == null) {
+        mRandomAccessFile = new RandomAccessFile(mFilePath.toString(), "rw");
+        mCurrentOffset = 0;
+      }
+      if (ClientIOOperation.isRead(mParameters.mOperation) && mParameters.mReadRandom) {
+        if (mParameters.mReadRandom) {
```

Comment:
redundant double check of `mParameters.mReadRandom`

Reply:
Thanks